research/alert_descriptions/index.md

```diff
@@ -0,0 +1,141 @@
+
+This could be a hint at what experts do and what information novices need to see on the 'View Details' page. Finally, the theme, "*information gathering*" indicated that users who didn't have enough information would resort to web searches, documentation searches, or support. Perhaps this is where a user who doesn't have the 'View Details' page meeting their needs would end up.
+
+*Quote from participant*
+
+**"I have not experienced this yet.  However, I generally can find answers through access.redhat.com, generally speaking"**
+
+If we could create a more robust path for users to resolve alerts within the interface, this could reduce the amount of support cases related to certain problematic alerts.
+
+## Conclusions
+
+Alert confidence varies widely among users, but it's often related to lack of experience with a certain resource. Providing more description or information around alerts related to less well-known resources could aid the user when troubleshooting.
+
+'View Details' is the most likely place someone will click when resolving an alert. This is typically motivated when the user does not know where they need to go already to resolve the issue. The 'View Details' page should support a type of user who is seeking out background information and troubleshooting strategies related to possible root causes.
+
+## Appendix
+
+### Alerts used in this study
+
+- **KubePodNotReady** Pod myproject/frontend-1 has been in a non-ready state for longer than 15 minutes.
+- **KubeDeploymentReplicasMismatch Deployment** myproject/frontend-1 has not matched the expected number of replicas for longer than 15 minutes.
+- **KubeAPIErrorsHigh** API server is returning errors for 90% of requests.
+- **MachineWithNoRunningPhase**  machine worker-us-east-1a is in unknown phase
+- **TargetDown** 50% of the job distributor targets in myproject namespace are down.
+- **KubeDaemonSetRolloutStuck** Only 25% of the desired Pods of DaemonSet myproject/frontend are scheduled and ready.
+- **KubePodCrashLooping** Pod myproject/python-1 container python is restarting 36 times / 5 minutes.
+- **KubeletDown** Kubelet has disappeared from Prometheus target discovery.
+- **CPUThrottlingHigh**  50% throttling of CPU in namespace myproject for container nodejs in pod nodejs-2.
+- **etcdHighCommitDurations** etcd cluster "etcd": 99th percentile commit durations 100s on etcd instance cluster-health.
+- **ClusterOperatorDegraded** Cluster operator insights has been degraded for 10 mins. Operator is degraded because Unable to report: gateway server reported unexpected error code: 415, and cluster upgrades will be unstable.
+
+### Analysis
+
+Plots and quanititative analyses were conducted in R using `tidyverse` and `lmerTest`. The repository for those analyses can be found [here](https://github.com/carljpearson/openshift_alerting), alongside more detailed plots.
```

Comment:
```suggestion
Plots and quantitative analyses were conducted in R using `tidyverse` and `lmerTest`. The repository for those analyses can be found [here](https://github.com/carljpearson/openshift_alerting), alongside more detailed plots.
```

I wonder if this is R work is worth highlighting more by including this sentence somewhere toward the top instead of the bottom.